src/tox_lsr/config_files/tox-default.ini

```diff
@@ -461,13 +461,33 @@ allowlist_externals = podman
     rm
 setenv =
     {[testenv]setenv}
-    GITHUB_OUTPUT = /tmp/mdl.log
```

Comment:
https://hub.docker.com/r/avtodev/markdown-lint
this markdown-lint has an option to set INPUT_OUTPUT=/foo but in this case no output is printed to console, which is unwanted.